beacon/validator/block_builder.go

```
@@ -298,7 +291,7 @@ func (s *Service) buildBlockBody(
 	// Dequeue deposits from the state.
 	depositIndex, err := st.GetEth1DepositIndex()
 	if err != nil {
-		return ErrNilDepositIndexStart
```

we silence the error here, which I do not like. Improved the error handling